bin/cucumber

```diff
@@ -15,7 +15,7 @@ ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile',
 bundle_binstub = File.expand_path('bundle', __dir__)
 
 if File.file?(bundle_binstub)
-  if File.read(bundle_binstub, 300) =~ /This file was generated by Bundler/
```

Comment:
Should we be fixing things in autogenerated files we don't own? Will we get in a loop of whenever we update binstubs we have to autocorrect them?

Should we instead exclude bin/ from rubocop?